### PR TITLE
Merge

### PR DESCRIPTION
**Pull request by isamitre**
so my podmates have style.css and my file organization

### CODE REVIEW

File: src/main/java/com/google/sps/data/Recipe.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  private long id;
+  private String recipeName;
+  private long authorID;
+  private String imageUrl;
+  private ArrayList<String> tags;
+  private ArrayList<String> ingredients;
+  private ArrayList<String> steps;
+  private int popularity;
+
+  public Recipe(String recipeName, ArrayList<String> tags, ArrayList<String> ingredients, ArrayList<String> steps){
```
Comment (ez-may) [inline src/main/java/com/google/sps/data/Recipe.java]: Nit: space between ) and {

File: src/main/java/com/google/sps/data/Recipe.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    this.recipeName = recipeName;
+    this.tags = tags;
+    this.ingredients = ingredients;
+    this.steps = steps;
+    popularity = 0;
+  }
+
+  public void setImageUrl (String imageUrl) {
```
Comment (ez-may) [inline src/main/java/com/google/sps/data/Recipe.java]: Nit: remove extra space between method name and parenthesis.

File: src/main/webapp/css/style-userpage.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  height: 100%; /* Full height */
+  left: 0;
+  overflow: auto; /* Enable scroll if needed */
+  padding-top: 100px;
+  position: fixed; /* Stay in place */
+  top: 0;
+  width: 100%; /* Full width */
+  z-index: 1; /* Sit on top */
+}
+#recipe-form {
```
Comment (ez-may) [inline src/main/webapp/css/style-userpage.css:14]: Can you include a line between your CSS rules? Ref: [CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation)

File: src/main/webapp/css/style.css [hunk 4]
```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```
Comment (ez-may) [inline src/main/webapp/css/style.css:4]: Can you add a white space here?

File: src/main/java/com/google/sps/servlets/NewRecipeServlet.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.images.ImagesService;
+import com.google.appengine.api.images.ImagesServiceFactory;
+import com.google.appengine.api.images.ServingUrlOptions;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
```
Comment (ez-may) [inline src/main/java/com/google/sps/servlets/NewRecipeServlet.java]: Can you try to follow and alphabetical order for your imports?

**Review by ez-may — APPROVED** [review]

File: src/main/java/com/google/sps/data/Recipe.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  public void setImageUrl(String imageUrl) {
+    this.imageUrl = imageUrl;
+  }
+
+  public void setID(long id) {
+    this.id = id;
+  }
+  public void setAuthorID(long id) {
```
Comment (oturner24) [inline src/main/java/com/google/sps/data/Recipe.java:37]: nit: add a space between methods (here and below)

File: src/main/webapp/css/style.css [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+.content {
+  max-width: 1000px;
+  width: 80%;
+  margin: 0 auto;
+  min-width: 300px;
+  padding: 0 20px 20px;
```
Comment (oturner24) [inline src/main/webapp/css/style.css]: nit: alphabetize

File: src/main/webapp/pages/UserPage.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+            <label for="ingredients" required>Ingredients:<br><i>Each ingredient should be on its own line.</i></label><br>
+            <textarea id="ingredients" name="ingredients" rows="4"></textarea>
+            <br><br>
+            <label for="steps" required>Steps:<br><i>Each step should be on its own line. No need to number them.</i></label><br>
+            <textarea id="steps" name="steps" rows="4"></textarea>
+            <br><br>
+            <label for="privacy">Choose a privacy setting:</label>
+            <select id="privacy" name="privacy">
+            <option value="private">private</option>
+            <!-- <option value="public">public</option> -->
```
Comment (oturner24) [inline src/main/webapp/pages/UserPage.html:47]: Should this be un-commented? Or is it not implemented yet?

**Review by oturner24 — APPROVED** [review]

File: src/main/webapp/pages/UserPage.html (hunk 8 above)
Comment (isamitre) [reply src/main/webapp/pages/UserPage.html:47]: it's not implemented yet

**Pull request merged**